pontoon/base/templates/user.html

```diff
@@ -1,4 +1,4 @@
-{% extends "landing.html" %}
+\{% extends "landing.html" %}
```

Comment:
Whoops :D added to fix list.

On Tue, Oct 6, 2015 at 5:27 PM, Matjaž Horvat <notifications@github.com>
wrote:

> In pontoon/base/templates/user.html
> <https://github.com/mozilla/pontoon/pull/201#discussion_r41279225>:
>
> > @@ -1,4 +1,4 @@
> > -{% extends "landing.html" %}
> > +\{% extends "landing.html" %}
>
> Whoops?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/pontoon/pull/201/files#r41279225>.
>



-- 
jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.